src/ophys_etl/modules/segmentation/merge/metric.py

```diff
@@ -0,0 +1,198 @@
+"""
+This module contains the code that roi_merging.py uses to calculate the
+merger metric for each candidate merger
+"""
+from typing import List, Tuple, Dict
+import numpy as np
+import multiprocessing
+import multiprocessing.managers
+from ophys_etl.modules.segmentation.merge.utils import (
+    _winnow_process_list)
+
+from ophys_etl.modules.segmentation.merge.roi_time_correlation import (
+        calculate_merger_metric)
+
+from ophys_etl.modules.segmentation.merge.characteristic_timeseries import (
+    CharacteristicTimeseries)
+
+
+def _calculate_merger_metric(
```

Comment:
this name is very close to `calculate_merger_metric`
could it be `calculate_merger_metrics` or something to distinguish more?

Reply:
maybe `_get_merger_metric_from_pairs_job`?

Reply:
done